mvpa/cosmo_classify_lda.m

```diff
@@ -138,7 +142,7 @@
                                 'in training']);
                 end
                 nfirst=n; % keep track of number of samples
-            elseif nfirst~=n
+            elseif opt.check_data & nfirst~=n
```

Comment:
This option is somewhat misleading, because if check_data is disabled, then the output can be *wrong* (since there is no support for variable class frequencies). Do we really need this option? In other words, in a typical use case, wouldn't it be better to have balanced training sets? If we really /do/ need it, can it be renamed to 'unbalance_with_invalid_prior_ok', or something similarly long-and-dangerously-sounding so that the novice user is not using this option by accident?